docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+## Overview
+
+Bitcoin is a peer-to-peer digital currency and network. Bitcoins can be sent and received through
+the internet, and aren't centrally controlled by anyone. Unlike using a bank, this means nobody
+can seize, block or take control of your money. Instead, Bitcoin is run and verified collectively by
+its users, as a peer-to-peer network. Let's get started and explain the history behind Bitcoin
+and how it works.
+
+## Background and transactions
+
+Bitcoin was first described in the Bitcoin whitepaper, by Satoshi Nakamoto, its first developer.
+Before Bitcoin, digital currencies and payment systems all relied on trust, by either trusting a
```

Comment:
LINE 4: Change "the internet, and aren't centrally controlled by anyone." to "the internet."
Lets rewrite the next paragraph to suggest that bitcoins are a "cryptological bearer asset" that the user is the owner of while in possession of the asset via a Bitcoin wallet. Again this will add to the confidence of an average user, yet suggests that the user is charged with responsibility of maintaining their assets (for better or worse, ignoring third party custodians for now).